modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerRecoveryTest.java

```diff
@@ -99,6 +101,27 @@ void testRecoveryCatalogVersionTimestamps() throws Exception {
         assertThat(catalogManager.activeCatalogVersion(time1), equalTo(catalogVersion1));
     }
 
+    @Test
+    void testRecoveryIndexCreationCatalogVersion() throws Exception {
+        createAndStartComponents();
+
+        assertThat(catalogManager.execute(simpleTable(TABLE_NAME)), willCompleteSuccessfully());
+        assertThat(catalogManager.execute(simpleIndex(TABLE_NAME, INDEX_NAME)), willCompleteSuccessfully());
+
+        int expCreationCatalogVersion = catalogManager.latestCatalogVersion();
+
+        int indexId = catalogManager.index(INDEX_NAME, clock.now().longValue()).id();
```

Comment:
I believe there's a `clock.nowLong` method

Reply:
Fix it